docs/getting-help/getting-help.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Getting Help with Prophecy
+sidebar_position: 3
+id: getting-help-with-prophecy
+description: Getting Help with Prophecy
+tags: []
+---
+
+At Prophecy, we're dedicated to ensuring you have the support you need, whether you're exploring our platform during a trial or you're one of our valued customers.
+
+## For Trial Users
+
+- **Get Trial Support in Slack:** Have questions during your trial? Join the conversation in the #support channel on our Slack community. Our dynamic community is here to assist you. [Join #support on Slack](https://prophecy-io-support.slack.com/archives/C01P1PD7JJY).
+- _If you're a Trial customer and unable to signup for our Slack Community, please reach out to Sales@Prophecy.io for quick assistance._
+
+## For Existing Customers
+
+- **Log a Ticket in the Support Portal:** For our existing customers, the Customer Support Portal is your hub for logging tickets and getting personalized assistance. Log in to the Support Portal [here](https://prophecy.zendesk.com/).
+- _Trouble Logging In? If you're encountering issues accessing the Customer Support Portal, don't worry. Please reach out to Support@Prophecy.io, and we'll get you back on track!_
```

Comment:
we can say if existing customers are facing problems, there are 2options:
1. raise a ticket in support portal.
2. reach out to support@prophecy.io (email in small case letters)